pkg/vault/contracts/ProtocolFeeController.sol

```diff
@@ -132,15 +135,15 @@ contract ProtocolFeeController is
 
     // Validate the yield fee percentage against the maximum.
     modifier withValidYieldFee(uint256 newYieldFeePercentage) {
-        if (newYieldFeePercentage > _MAX_PROTOCOL_YIELD_FEE_PERCENTAGE) {
+        if (newYieldFeePercentage > MAX_PROTOCOL_YIELD_FEE_PERCENTAGE) {
             revert ProtocolYieldFeePercentageTooHigh();
         }
         _ensureValidPrecision(newYieldFeePercentage);
         _;
     }
 
     modifier withValidPoolCreatorFee(uint256 newPoolCreatorFeePercentage) {
-        if (newPoolCreatorFeePercentage > FixedPoint.ONE) {
+        if (newPoolCreatorFeePercentage > MAX_CREATOR_FEE_PERCENTAGE) {
```

Comment:
This is below the max fee percentage, as the absolute max fee should accommodate max pool creator with max protocol fee percentage. Added test below for that specific condition.